content/resources/help-centre/converting/convert-r/index.md

```diff
@@ -1,63 +1,279 @@
+[warbleR](https://cran.r-project.org/web/packages/warbleR/index.html) package. 
+
+``` r
+library(warbleR)
+mp32wav(path = "data/", dest.path = "output/")
+```
+{{% hint warning %}}
+**Warning:**  
+The mp32wav function relies on the tuneR package. Sometimes, this function does
+not work, and RStudio may abort. Although this bug should be fixed in future
+versions of tuneR, if this happens to you, alternative conversion methods are
+available (see our [ffmpeg]({{% ref"ffmpeg"%}}) or Audacity (todo) help pages).   
+{{% /hint %}}
 
-## Working with .flac
+## Importing and converting .wac files
+The utility to read the Wildlife Acoustics proprietary compressed format
+(`.wac`) is provided by the
+[bioacoustics](https://cran.r-project.org/web/packages/bioacoustics/index.html)
+package, with their `read_wac` function. See the example below on using the
+`read_wac` function directly using the bioacoustics package. This function has
+the useful option of converting your `.wac` files into `.wav` using the
+`write_wav` setting. 
+
+``` r
+library(bioacoustics)
+# writes wav files to output folder of your choosing
+sound_wac <- read_wac(file = "data/recording_20170716_230503.wac", write_wav = "output/")
+```
+``` r
+sound_wac
+## Wave Object
+##	Number of Samples:      230144
+##	Duration (seconds):     0.6
+##	Samplingrate (Hertz):   384000
+##	Channels (Mono/Stereo): Mono
+##	PCM (integer format):   TRUE
+##	Bit (8/16/24/32/64):    16 
+```
+## Importing and converting .flac
 tuneR does not read `.flac`, but there are packages which can help you to
-work with `.flac`. 
+work with `.flac`, and perform conversions between `.flac` and `.wav`
 
 ### seewave
-seewave provides an integration with a software called
+The [seewave](https://cran.r-project.org/web/packages/seewave/index.html)
+package provides an integration with a software called
 [FLAC](https://xiph.org/flac/index.html) that converts `.flac` files into `wav`,
 by using the function `wav2flac`. The function actually converts both ways
 (`.flac` to `.wav`, `.wav` to `.flac`). Before seewave can convert `.flac`, you
 will need to install the FLAC software. See our [FLAC]({{< ref"FLAC">}}) help
-page for more information. 
+page for more information. Further details on the `wav2flac` function can be
+found [here](https://rdrr.io/cran/seewave/man/wav2flac.html). 
 
-#### Example    
+Take note of the `exename` and `path2exe` settings in the examples below. If you
+are using windows, path2exe should specify the folder which contains flac.exe.
+If you are using MacOS, the directory that flac is installed should be listed in
+your `$PATH` variable for the function to work. 
+
+{{% tabs "seewave_flac" %}}
+{{% tab "Windows" %}} 
+``` r
+library(seewave)
+# reverse = TRUE covnerts a flac to a wav file
+ wav2flac(
+         file = "data/20200117T000000+1000.flac",
+         reverse = TRUE,
+         overwrite = FALSE,
+         exename = "flac.exe",
+         path2exe = "C:/ProgramData/chocolatey/bin")
```

Comment:
it's great choco is used to install flac, however once it does flac should be on `PATH` - is it actually necessary to specify the path here?

also this path makes an assumption on how ffmpeg was installed, it can be installed other ways... it'd be best if we didn't have to specify this at all